foundations/installations/installation_overview.md

```diff
@@ -55,19 +61,15 @@ If you don't have an Apple computer, you are likely using Windows. Don't worry!
 
 Modifying or dual booting a computer to work with the tools you'll need will make it much easier to start programming, can help create a distraction-free environment, and will look good on your resume. Take a deep breath, and let's look at your options.
 
-Still not convinced? Here are a few great reasons to install Linux: 
+Still not convinced? Here are a few great reasons to install Linux:
```

Comment:
The line below this could possibly be changed to:

```md
**Tested** - We've tested our directions with macOS, Ubuntu (and official Ubuntu flavors) and WSL2. We did the research so that you can get tools installed with as few issues as possible, getting you to coding sooner. Time spent wrestling with your operating system is time taken from learning how to code.
```

Reply:
Done